_src/_posts/2018-01-04-importance-of-serving-ads.markdown

```diff
@@ -124,51 +120,43 @@ These relationships were created from the data, yet we could not explicitly give
 
 As a second step, we needed to define business metrics that are relevant for our customers. We will have that different frequency cappings would produce different levels of them, because of how the relationships \\(Imp\\), \\(Conv\\) and \\(Cl\\) vary along cap levels.
```

Comment:
"We will have that" doesn't make sense.
Simply:  Different frequency cappings would produce different levels of them ...